src/averageAges.js

```diff
@@ -14,14 +13,31 @@
  *
  * @return {number}
  */
+
+function calculateAverageAge(person) {
+  const AverageSumOfAllPeople = person.reduce((acc, curr) =>
```

Comment:
Змінні починаємо з великої буквою для:
1.  функцій-конструкторів
2. класів
```suggestion
  const liveDurationSum = person.reduce((acc, curr) =>
```